discover/portal/articles/130-forms/01-whats-new.markdown

```diff
@@ -0,0 +1,81 @@
+# What's New with Liferay Forms
+
+Liferay Forms is evolving. Features that were new in the last (and first)
+version have been enhanced, and new features added. This article compiles the
+prominent changes and additions. More details on these features are found in the
+other articles of this section.
+
+**Form Rules**
+: Set dynamic form behavior by specifying conditions (*if this happens in field
+X*) and actions (*then do this: X*). Available actions include Require,Enable,
+
+![Figure x: Form Rules trigger an action in response to some condition.](../../images/forms-rule-development.png)
+
+**Form Translation**
+: Provide translations of the same form into the platform's supported languages.
+Form Users select the language they'd like to see.
+
+![Figure x: Translate a Form into a supported language.](../../images/forms-translation.png)
+
+**New Fields and Properties**
+: File Upload fields let logged in Users upload documents tot he Form. Numeric
+fields validate that input is numeric. Grid fields let Users select from options
+laid out in rows and columns. One selection can be made per row. 
+
+![Figure x: File Upload, Grid, and Numeric fields are new.](../../images/forms-sidebar.png)
+
+**Element Sets**
+: Create reusable Form fragments that can be used to quickly populate new Forms
+with common fields and configurations.
+
+![Figure x: Reusable Element Sets are great](../../images/forms-element-set.png)
+
+**Autosave**
+: By default, a Form is autosaved every minute (without the Form builder
+clicking *Save*). The duration is configurable in Control Panel &rarr;
+Configuration &rarr; System Settings &rarr; Forms. Autosaves do not propagate
+to published Forms. 
+
+![Figure x: ](../../images/forms-autosave-interval.png)
+
+**Autocomplete Text Fields**
+: Using a Data Provider or manually typed options, configure a Text field to
+offer Users automcompleted values to choose from as they type in the Text field.
+Set up autocomplete in Text field's Properties.
+
+![Figure x: A User beings typing in a Text field and is pleasantly surprised by a list of
+autocomplete options.](../../images/forms-autocomplete.png)
+
+**Form Success Page**
+: Add a Form Page that displays after a successful Form submission.
+
+![Figure x: A Success Page provides feedback to the User that the form
+Submission was successful.](../../images/forms-success-page.png)
+
+**Duplicate Forms and Form Fields**
+: Duplicate a Form to avoid building a similar one from scratch. Duplicate Form
+Fields if you need multiple similarly configured fields.
+
+![Figure x: Duplicate a Form to get a head start on your next, similar Form.](../../images/forms-duplicate.png)
+
+**Validate Text Field Entries**
```

Comment:
Actually, this is a feature already available on 7.0. The real new feature in 7.1 is the possibility to use regular expressions when the field _If input_ has the value _Does not match_. 😉